suite-native/module-home/src/components/BiometricsBottomSheet.tsx

```diff
@@ -0,0 +1,195 @@
+    getIsBiometricsFeatureAvailable,
+} from '@suite-native/biometrics';
+import { Translation, useTranslate } from '@suite-native/intl';
+import { Icon } from '@suite-common/icons/src';
+import { useToast } from '@suite-native/toasts';
+import { selectHasDecidedBiometrics, setHasDecidedBiometrics } from '@suite-native/module-settings';
+
+const SHOW_TIMEOUT = 1500;
+
+const cardStyle = prepareNativeStyle(utils => ({
+    alignItems: 'center',
+    textAlign: 'center',
+    borderRadius: utils.borders.radii.medium,
+    backgroundColor: utils.colors.backgroundSurfaceElevation1,
+    paddingVertical: utils.spacings.large,
+    marginBottom: utils.spacings.large,
+    marginTop: utils.spacings.extraLarge,
+    gap: utils.spacings.large,
+}));
+
+const imageWrapperStyle = prepareNativeStyle(utils => ({
+    padding: 12,
+    borderRadius: utils.borders.radii.round,
+    backgroundColor: utils.colors.backgroundSurfaceElevation2,
+    color: utils.colors.iconPrimaryDefault,
+    width: 88,
+    height: 88,
+}));
+
+const textContentStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.small,
+}));
+
+const buttonWrapperStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.medium,
+    paddingHorizontal: utils.spacings.small,
+}));
+
+type BiometricsBottomSheetProps = {
+    onClose?: (isVisible: boolean) => void;
+};
+
+export const BiometricsBottomSheet = ({
+    onClose = (_: boolean) => {},
+}: BiometricsBottomSheetProps) => {
+    const { applyStyle } = useNativeStyles();
+    const { translate } = useTranslate();
+    const { showToast } = useToast();
+    const hasDecidedBiometrics = useSelector(selectHasDecidedBiometrics);
+    const { isBiometricsOptionEnabled } = useIsBiometricsEnabled();
+
+    const [isVisible, setIsVisible] = useState(false);
+    const [biometricsTypes, setBiometricsTypes] = useState<AuthenticationType[] | undefined>(
+        undefined,
+    );
+
+    const isFace = biometricsTypes?.filter(x =>
+        [AuthenticationType.FACIAL_RECOGNITION, AuthenticationType.IRIS].includes(x),
+    );
+
+    const iconName = useMemo(() => {
+        if (isFace) {
+            return 'faceId';
+        }
+
+        return 'fingerprint';
+    }, [isFace]);
+
+    const title = useMemo(() => {
+        if (biometricsTypes?.length === 0) {
+            return <Translation id="moduleHome.biometricsModal.title.unknown" />;
+        }
+        if (isFace) {
+            return <Translation id="moduleHome.biometricsModal.title.faceId" />;
+        }
+        if (Platform.OS === 'android') {
+            return <Translation id="moduleHome.biometricsModal.title.fingerprint" />;
+        }
+        return <Translation id="moduleHome.biometricsModal.title.touchId" />;
+    }, [biometricsTypes, isFace]);
+
+    useEffect(() => {
+        const getSupportedTypes = async () => {
+            const biometricsTypesAvailable = await getSupportedBiometricsTypes();
+            if (biometricsTypesAvailable.length === 0) {
```

Comment:
This `if` could be removed, you can pass empty array to `setBiometricsTypes`

Reply:
removed in d385a4d